_data/projects/Python-ChatBot.yml

```diff
@@ -0,0 +1,11 @@
+name: Python-ChatBot
+desc: This is a program which perform actions on user text commands such as opening apps,performing google search and playing games.(A mini assitant)
+site: https://github.com/Aswin-Poovadan/Python-ChatBot
+tags:
+  - python3
+  - chat-bot
+  - cli
+  - text-command
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/Aswin-Poovadan/Python-ChatBot/labels/up-for-grabs
```

Comment:
@Aswin-Poovadan Can you please open up some issues at your repo with this label marked so that other can find how to contribute?

Reply:
Yeah sure ,will open up the issues

Reply:
Added the issues .Please check this